electron/main.js

```diff
@@ -109,12 +109,17 @@ function createWindow () {
 	if ( config.get('user_agent').length > 0 ) mainWindow.webContents.setUserAgent( config.get('user_agent') );
 
 	if ( !config.get('start_minimized') && config.get('maximized') ) mainWindow.maximize();
-	if ( config.get('window_display_behavior') !== 'show_trayIcon' && config.get('start_minimized') ) {
-		// Wait for the mainWindow.loadURL(..) and the optional mainWindow.webContents.openDevTools()
-		// to be finished before minimizing
-		mainWindow.webContents.once('did-finish-load', function(e) {
-			mainWindow.minimize();
-		});
+	if (config.get('start_minimized')){
```

Comment:
Thanks, that's a great idea. But, after some thinking, we decided that we only want this behavior when it's set in "Show tray icon".

I modified your code to get more in tune with the way that Rambox is coded, please make this changes and I will be glad to merge this PR

```
	// Wait for the mainWindow.loadURL(..) and the optional mainWindow.webContents.openDevTools()
	// to be finished before minimizing
	config.get('start_minimized') && mainWindow.webContents.once('did-finish-load', () => config.get('window_display_behavior') === 'show_trayIcon' ? mainWindow.hide() :  mainWindow.minimize());
 ```

Also, please don't remove the comments.

Once merged your PR will be part of our next version. Thanks for your contribution to the project!